Src/IronPython/Runtime/Bytes.cs

```diff
@@ -60,7 +64,17 @@ public class Bytes : IList<byte>, IEquatable<Bytes>, ICodeFormattable, IExpressi
             _bytes = StringOps.encode(context, unicode, encoding, "strict").UnsafeByteArray;
         }
 
-        private static readonly Bytes[] oneByteBytes = Enumerable.Range(0, 256).Select(i => new Bytes(new byte[] { (byte)i })).ToArray();
+        private static readonly IReadOnlyList<Bytes> oneByteBytes;
+
+        static Bytes() {
+            var barr256 = new Bytes[256];
```

Comment:
Not exactly sure what's gained by using this instead of the one-liner above?

Reply:
In my attempts to resolve the ambiguity calling the constructor with a `byte[]` type argument, one of the things I tried was to make `Bytes(byte[])` public. Then neither cast to `IList` nor to `Span` would be necessary.

But this would mean that this constructor had to copy the input, so I tried to eliminate the need for a non-copying constructor. This static constructor is a left-over form that, which I kept around to give myself more freedom to experiment further, but at this point I am ready to give it up. In the next commit I will restore the one-liner.

Reply:
You can use `Bytes.Make` to avoid any ambiguity.